### PR TITLE
fix: multicall tests draft

### PR DESCRIPTION
**Pull request by 0xmikko**
- small changes in CF

### CODE REVIEW

File: contracts/test/helpers/CreditFacadeTestHelper.sol [hunk 1]
```diff
@@ -132,14 +132,14 @@ contract CreditFacadeTestHelper is TestHelper {
         vm.roll(block.number + 1);
     }
 
-    function multicallBuilder() internal pure returns (MultiCall[] memory calls) {}
+    function MultiCallBuilder() internal pure returns (MultiCall[] memory calls) {}
```
Comment (lekhovitsky) [inline contracts/test/helpers/CreditFacadeTestHelper.sol]: is it by intent or a result of automatic renaming?

File: contracts/interfaces/ICreditFacadeMulticall.sol [hunk 2]
```diff
@@ -28,7 +28,7 @@ uint256 constant INCREASE_DEBT_WAS_CALLED = 2 ** 193;
 uint256 constant EXTERNAL_CONTRACT_WAS_CALLED = 2 ** 194;
 
 // Pay bot permisson is set there because it's used for bots only
```
Comment (lekhovitsky) [inline contracts/interfaces/ICreditFacadeMulticall.sol]: probably worth updating the comment

**Review by lekhovitsky — APPROVED** [review]

File: contracts/test/helpers/CreditFacadeTestHelper.sol (hunk 1 above)
Comment (0xmikko) [reply contracts/test/helpers/CreditFacadeTestHelper.sol]: Fixed

File: contracts/interfaces/ICreditFacadeMulticall.sol (hunk 2 above)
Comment (0xmikko) [reply contracts/interfaces/ICreditFacadeMulticall.sol]: fixed

**Pull request merged**